src/backend/api_app.py

```diff
@@ -6,14 +10,53 @@
 app = Flask(__name__)
 api = Api(app)
 
+UPLOAD_FOLDER = 'static/uploads/'
+app.secret_key = "secret key"
+
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+
+
 class RalphLaurenFeedPredictions(Resource):
-	def get(self):
+    def get(self):
+        
+        upload_file()
+        
 		# Will eventually make batch predictions
-		pass
+        pass
 
 
 api.add_resource(RalphLaurenFeedPredictions, '/ralph_lauren_feed_predict')
 
-if __name__ == '__main__':
-	app.run()
+
+
+
+# Allowed file type extensions
+
+ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg', 'gif'])
+
+def allowed_file(filename):
+
+    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+# Code to run uploaded file and store it in a folder
+
+@app.route('/uploader', methods = ['POST'])
```

Comment:
Should be a GET request, since we need to pass image to API, could be a POST if you wanted to implement a SOAP API, but right now this is moving in the direction of Restful